lib/pure/stats.nim

```diff
@@ -320,6 +320,22 @@ proc `+=`*(a: var RunningRegress, b: RunningRegress) =
 {.pop.}
 {.pop.}
 
+
+runnableExamples:
+  static:
+    block:
+      var statistics: RunningStat  ## Must be "var"
+      statistics.push(@[1.0, 2.0, 1.0, 4.0, 1.0, 4.0, 1.0, 2.0])
+      doAssert statistics.n == 8
+      doAssert statistics.mean() is SomeFloat
```

Comment:
@juancarlospaco instead of these weak tests how about using `==~` which you can define as:
```nim
proc `==~`(x, y: float): bool = (abs(x-y) < 1e-9) # as in std/math
```
(note: ideally should be implemented in relative terms, but good enough for this test)